palindromeChecker/index.html

```diff
@@ -6,19 +6,21 @@
 
         <title>Palindrome Checker</title>
 
-        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" href="../styles/projectsStyle.css">
     </head>
     <body>
         <div class="container">
             <h1 class="title">PALINDROME CHECKER</h1>
 
-            <div>
+            <div class="container-block">
                 <div>
-                    <p class="text">Enter your word:<input type="text" id="input"></p>
+                    <form id="myForm">
```

Comment:
You should add form validation.
1. The user should not be able to enter something besides numbers.
2. The user should not be able to enter a number less than 1 and greater than 3999. (show a validation error or use another type of input for that)